source/_docs/dns.md

```diff
@@ -1,12 +1,161 @@
 ---
-title:  DNS Providers
-layout: doc
-tags: [domains]
-permalink: docs/:basename/
-use: [docs_tags]
+title: DNS
+description: Learn the basics of DNS.
+tags: [dns]
+categories: []
+contributors: alexfornuto
 ---
-{% for doc in data.docs_tags.providers %}
```

Comment:
Consider keeping the list of DNS provider specific docs as it's own section towards the top of the page so people can easily find instructions for their provider.

Reply:
@rachelwhitton what about the reference on Line 13?

Reply:
Not seeing the value in having that list on it's own page - suggest deleting dns-providers since this page is essentially the "parent" page for that one. We should just provide the list here, yeah?